templates/repo/pinned_repos.tmpl

```diff
@@ -0,0 +1,76 @@
+{{if .PinnedRepos}}
```

Comment:
Isn't this template slightly misplaced and should rather belong into `shared`?
I mean, it displays a bunch of repos and not something specific about a single repo, and both orgs and users should be able to use the same template.

Reply:
Moved